front/src/applications/operationalStudies/views/ManageTrainSchedule.tsx

```diff
@@ -52,40 +50,84 @@ export default function ManageTrainSchedule({ setDisplayTrainScheduleManagement
       }),
     }
   );
+  const tabRollingStock = {
+    title: (
+      <>
+        <img src={rollingStockPic} alt="rolling stock" height={24} />
+        <span className="ml-2">{t('tabs.rollingStock')}</span>
+      </>
+    ),
+    withWarning: rollingStockID === undefined,
+    label: t('tabs.rollingStock'),
+    content: <RollingStockSelector />,
+  };
+
+  const tabPathFinding = {
+    title: (
+      <>
+        <img src={pahtFindingPic} alt="path finding" height={24} />
+        <span className="ml-2">{t('tabs.pathFinding')}</span>
+      </>
+    ),
+    withWarning: pathFindingID === undefined,
+    label: t('tabs.pathFinding'),
+    content: (
+      <div className="row no-gutters">
+        <div className="col-xl-6 pr-xl-2">
+          {mustUpdatePathfinding !== undefined && <Itinerary mustUpdate={mustUpdatePathfinding} />}
```

Comment:
Better to initialize mustUpdatePathfinding to false 
```suggestion
          <Itinerary mustUpdate={mustUpdatePathfinding} />
```

Reply:
Yes, was not my work but I corrected it anyway ;)